roles/openshift_service_catalog/tasks/wire_aggregator.yml

```diff
@@ -138,6 +143,10 @@
       value: [X-Remote-Group]
     - key: authConfig.requestHeader.extraHeaderPrefixes
       value: [X-Remote-Extra-]
+    - key: assetConfig.extensionScripts
+      value: [/etc/origin/master/openshift-ansible-catalog-console.js]
+    - key: kubernetesMasterConfig.apiServerArguments.runtime-config
+      value: [apis/settings.k8s.io/v1alpha1=true]
```

Comment:
Does this [] notation ensure that it's only an additive thing or would this overwrite whatever they have now? If it's overwriting I'm fine with this for now but we'll want to make sure it's additive as a followup.